src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -134,6 +134,12 @@ p5.RendererGL.prototype.vertex = function(x, y) {
         u /= this._tex.width;
         v /= this._tex.height;
       }
+    } if (
+      (this.userFillShader !== undefined ||
```

Comment:
Just a minor style comment: I think we can take out the extra set of brackets around the if condition.

Reply:
Good point. Let me make the changes!